index.html

```diff
@@ -9,17 +9,21 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <script src="scripts.js"></script>
   </head>
   <body>
     <main class="container">
       <header class="header">
-        <a class="logo">John Doe</a>
+        <a id="header-logo" class="logo active-logo">Camilo Vela</a>
         <nav class="nav-button">
-          <img class= "normal-button" src="images/Enabled.png" alt="normal_button">
-          <a href="#" class="header-link">Portfolio</a>
-          <a href="#" class="header-link">About</a>
-          <a href="#" class="header-link">Contact</a>
+          <div class="links-container active-menu" id="links-container">
+            <a href="#recent-works" class="header-link">Portfolio</a>
+            <a href="#about-me" class="header-link">About</a>
+            <a href="#contact-us-container" class="header-link">Contact</a>
+          </div>
           <img class="mail-icon" src="images/icon_mail.png" alt="mail_icon">
+          <button class="hamburger-btn switch-btn" id="hamburger-btn" onclick="showMenu()">
+          </button>

```

Comment:
- [ ] Great job implementing the functionality of `nav-menu` 👌💯, but On the `desktop version` the hamburger menu button and the exit icon **should not be visible**. the nav-menu should work only on the `mobile version` 👌
- [ ] The `desktop-version` should look like your Figma design ( should have all the links in a row ), so kindly fix it 👌 

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/179665356-3e268d3e-88f9-44aa-8a52-ad051f4179b2.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/179665625-a6fce9f2-0d8d-4339-a28c-c000fab1ad0d.png" width="500" /> |

Reply:
![image](https://user-images.githubusercontent.com/106117541/179806154-de1132c1-db10-432e-a95a-0c0eb71ea8cd.png)